crates/teloxide-core/CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [pr851]: https://github.com/teloxide/teloxide/pull/851
 
+### Changed
+
+- `CallbackGame`, `ForumTopicClosed`, `ForumTopicReopened`, `GeneralForumTopicHidden`, `GeneralForumTopicUnhidden` and `WriteAccessAllowed` structures
+  are now defined as named (`struct S {}`) instead of unit (`struct S;`) in order to fix their deserialization
```

Comment:
Linking https://github.com/teloxide/teloxide/issues/874 would be great.

Reply:
Are you sure this is useful? #874 is not very descriptive...